### PR TITLE
py-pyqt5-*: update to 5.14.1; switch to PyPI

### PR DESCRIPTION
**Pull request by dliessi**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G11023
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?

Actually, `port lint` gives `Error: Line 112 repeats inclusion of PortGroup qmake5`, but the message is not correct: there are two inclusions of PortGroup qmake5 but in mutually exclusive conditionals, so only one is executed.

- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

I only tested py37-pyqt5 and py37-pyqt5-webengine.

### CODE REVIEW

File: python/py-pyqt5/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup               python 1.0
 
 name                    py-pyqt5
```
Comment (reneeotten) [inline python/py-pyqt5/Portfile]: to further simplify the portfile, please add ```python.rootname     PyQt5``` under the ```name``` field (you can then remove a few more lines below)

File: python/py-pyqt5/Portfile [hunk 2]
```diff
@@ -15,15 +15,11 @@ long_description        ${description}. The bindings \
...
-distname                PyQt5_gpl-${version}
-checksums               rmd160  3b68fe4e6c86e666145e71925e0226f4e2ab672a \
-                        sha256  54b7f456341b89eeb3930e786837762ea67f235e886512496c4152ebe106d4af \
-                        size    3162737
-
-livecheck.type          regex
-livecheck.url           https://www.riverbankcomputing.com/software/pyqt/download5
-livecheck.regex         >PyQt5_gpl-(\[0-9.\]*).tar.gz<
+master_sites            pypi:P/PyQt5/
+distname                PyQt5-${version}
```
Comment (reneeotten) [inline python/py-pyqt5/Portfile]: the ```master_sites``` and ```distname``` fields can be removed when setting ```python.rootname```

File: python/py-pyqt5/Portfile [hunk 3]
```diff
@@ -72,40 +68,35 @@ if {${subport} eq "${name}-common"} {
     destroot.destdir    DESTDIR=${destroot}
 
     if {[string first "webengine" ${subport}] != -1} {
-        version             5.13.1
+        version             5.14.0
         revision            0
```
Comment (reneeotten) [inline python/py-pyqt5/Portfile]: add ```python.rootname     PyQtWebEngine``` and remove ```master_sites```, ```distname``` and ```livecheck.name``` below

File: python/py-pyqt5/Portfile [hunk 4]
```diff
@@ -72,40 +68,35 @@ if {${subport} eq "${name}-common"} {
...
                             qtwebengine
 
-        livecheck.type      regex
-        livecheck.url       https://www.riverbankcomputing.com/software/pyqtwebengine/download
-        livecheck.regex     >PyQtWebEngine_gpl-(\[0-9.\]*).tar.gz<
-
+        livecheck.name      PyQtWebEngine
     } elseif {[string first "chart" ${subport}] != -1} {
-        version             5.13.0
+        version             5.14.0
```
Comment (reneeotten) [inline python/py-pyqt5/Portfile]: add ```python.rootname     PyQtChart``` and remove ```master_sites```, ```distname``` and ```livecheck.name``` below

**Comment by dliessi** [comment]
Done.

**Comment by pmetzger** [comment]
@reneeotten I presume you'll be merging this?

**Comment by reneeotten** [comment]
thanks @dliessi !

**Comment by pmetzger** [comment]
I think we should probably be getting rid of Travis; it rarely actually gives us useful information. Github's own CI system might be a good addition though.

**Pull request merged**